native/src/routeResultPreparation.cpp

```diff
@@ -1640,14 +1672,17 @@ void prepareTurnResults(RoutingContext* ctx, vector<SHARED_PTR<RouteSegmentResul
     addTurnInfoDescriptions(result);
 }
 
-vector<SHARED_PTR<RouteSegmentResult> > prepareResult(RoutingContext* ctx, vector<SHARED_PTR<RouteSegmentResult> >& result) {
-    combineWayPointsForAreaRouting(ctx, result);
-    validateAllPointsConnected(result);
-    splitRoadsAndAttachRoadSegments(ctx, result);
-    calculateTimeSpeed(ctx, result);
-    
-    prepareTurnResults(ctx, result);
-    return result;
+vector<SHARED_PTR<RouteSegmentResult> > prepareResult(RoutingContext* ctx,
+													  vector<SHARED_PTR<RouteSegmentResult> >& result) {
+	combineWayPointsForAreaRouting(ctx, result);
+	validateAllPointsConnected(result);
+	splitRoadsAndAttachRoadSegments(ctx, result);
+	for (int i = 0; i < result.size(); i++) {
+		filterMinorStops(result.get(i));
```

Comment:
Method ```get()``` does not exist fro std::vector. Please use the square braces like you would in Java's primitives array.